package.json

```diff
@@ -11,6 +11,7 @@
     "release:version-bump": "lerna version --force-publish=@apollo/query-planner-wasm",
     "release:start-ci-publish": "node -p '`Publish (dist-tag:${process.env.APOLLO_DIST_TAG || \"latest\"})`' | git tag -F - \"publish/$(date -u '+%Y%m%d%H%M%S')\" && git push origin \"$(git describe --match='publish/*' --tags --exact-match HEAD)\"",
     "postinstall": "npm run rustup-ensure && lerna run monorepo-prepare --stream && npm run compile",
+    "pretest": "npm run compile",
```

Comment:
Your commit message reminds me - the cached TS builds were problematic for us when they were first introduced. Maybe we can stop wiping that cache in our `clean` script?

Reply:
While I agree with it possibly being more reliable and stable than it was when it was first released, I think I would still prefer — sanity-wise — if `clean` really did a proper clean, including cached build artifacts.